### DISCUSSION TITLE
v9.0-(1403-5-30)

### DISCUSSION THREAD

**Discussion by GFW-knocker** [GFW-knocker/MahsaNG#71] [Announcements]
add utility for creating warp config
add wireguard noise parameters
add splithttp support
add Light/Dark Mode selection
update Core to 1.8.23
warp speed improvement
UI improvement
stability improvement

<hr /><em>This discussion was created from the release <a href='https://github.com/GFW-knocker/MahsaNG/releases/tag/v9.0-(1403-5-30)'>v9.0-(1403-5-30)</a>.</em>

**Comment by ASH512CH** [comment]
ورژن ویندوز هم بسازید

**Reply by Erkalin** [reply]
ورژن ویندوز و لینوکس هم بیارید

**Reply by GFW-knocker** [reply]
کلاینت مالتی پلتفرم در دست تهیه هست
زمان ریلیز نامشخص
